File: Programming Basics/First Steps in Coding/Lab/09_yard_greening.py
square_meters_landscaping = float(input())
square_meter_price = 7.61

sum = square_meters_landscaping * square_meter_price
discount = sum * 0.18
total_sum = sum - discount

print(f"The final price is: {total_sum} lv.")
print(f"The discount is: {discount} lv.")

File: Basics with JavaScript/01. First Steps In Coding/Lab/09. yardGreening.js
function yardGreening(input) {
    const priceSquareMeter = 7.61;    
    let metersForGreening = input[0]
    let totalPrice = metersForGreening * priceSquareMeter 
    let discount = totalPrice * 0.18
    let priceWithDiscount = totalPrice - discount
    console.log(`The final price is: ${priceWithDiscount} lv.`)
    console.log(`The discount is: ${discount} lv.`)
}

yardGreening(["550"])